.travis.yml

```diff
@@ -66,24 +69,30 @@ matrix:
       env: RUN_CHECKSTYLE=true
       script: mvn checkstyle:check
     - jdk: oraclejdk8
-      addons:
-        postgresql: "9.4"
+      sudo: required
+      dist: trusty
       env:
         - PG_VERSION=9.4
+        - XA=true
+        - REPLICATION=true
         - COVERAGE=Y
         - MCENTRAL=Y
     - jdk: openjdk7
-      addons:
-        postgresql: "9.4"
+      sudo: required
+      dist: trusty
       env:
         - PG_VERSION=9.4
         - MCENTRAL=Y
+        - XA=true
+        - REPLICATION=true
     - jdk: openjdk6
-      addons:
-        postgresql: "9.4"
+      sudo: required
+      dist: trusty
```

Comment:
-1 for converting lots of jobs to `sudo: required`. It will make Travis a lot slower